lib/iris/fileformats/grib/_grib1_load_rules.py

```diff
@@ -358,74 +250,16 @@ def add_bounded_time_coords(aux_coords_and_dims, grib):
-
-    if \
-            (grib.edition == 2) and \
-            (grib.typeOfFirstFixedSurface == 100) and \
-            (grib.typeOfSecondFixedSurface == 255):
-        aux_coords_and_dims.append((DimCoord(points=grib.scaledValueOfFirstFixedSurface/(10.0**grib.scaleFactorOfFirstFixedSurface), long_name="pressure", units="Pa"), None))
-
-    if \
-            (grib.edition == 2) and \
-            (grib.typeOfFirstFixedSurface == 100) and \
-            (grib.typeOfSecondFixedSurface != 255):
-        aux_coords_and_dims.append((DimCoord(points=0.5*(grib.scaledValueOfFirstFixedSurface/(10.0**grib.scaleFactorOfFirstFixedSurface) + grib.scaledValueOfSecondFixedSurface/(10.0**grib.scaleFactorOfSecondFixedSurface)), long_name="pressure", units="Pa", bounds=[grib.scaledValueOfFirstFixedSurface/(10.0**grib.scaleFactorOfFirstFixedSurface), grib.scaledValueOfSecondFixedSurface/(10.0**grib.scaleFactorOfSecondFixedSurface)]), None))
-
-    if \
-            (grib.edition == 2) and \
-            (grib.typeOfFirstFixedSurface in [105, 119]) and \
-            (grib.numberOfCoordinatesValues > 0):
-        aux_coords_and_dims.append((AuxCoord(grib.scaledValueOfFirstFixedSurface, standard_name='model_level_number', attributes={'positive': 'up'}), None))
-        aux_coords_and_dims.append((DimCoord(grib.pv[grib.scaledValueOfFirstFixedSurface], long_name='level_pressure', units='Pa'), None))
-        aux_coords_and_dims.append((AuxCoord(grib.pv[grib.numberOfCoordinatesValues//2 + grib.scaledValueOfFirstFixedSurface], long_name='sigma'), None))
-        factories.append(Factory(HybridPressureFactory, [{'long_name': 'level_pressure'}, {'long_name': 'sigma'}, Reference('surface_air_pressure')]))
-
     if grib._originatingCentre != 'unknown':
         aux_coords_and_dims.append((AuxCoord(points=grib._originatingCentre, long_name='originating_centre', units='no_unit'), None))
 
-    if \
-            (grib.edition == 2) and \
-            (grib.productDefinitionTemplateNumber == 1):
-        aux_coords_and_dims.append((DimCoord(points=grib.perturbationNumber, long_name='ensemble_member', units='no_unit'), None))
-
-    if \
-            (grib.edition == 2) and \
-            grib.productDefinitionTemplateNumber not in (0, 8):
-        attributes["GRIB_LOAD_WARNING"] = ("unsupported GRIB%d ProductDefinitionTemplate: #4.%d" % (grib.edition, grib.productDefinitionTemplateNumber))
-
-    if \
-            (grib.edition == 2) and \
-            (grib.centre == 'ecmf') and \
-            (grib.discipline == 0) and \
-            (grib.parameterCategory == 3) and \
-            (grib.parameterNumber == 25) and \
-            (grib.typeOfFirstFixedSurface == 105):
-        references.append(ReferenceTarget('surface_air_pressure', lambda cube: {'standard_name': 'surface_air_pressure', 'units': 'Pa', 'data': np.exp(cube.data)}))
-
     return ConversionMetadata(factories, references, standard_name, long_name,
```

Comment:
Why has loads of stuff been taken out of here? I can't really pass comment on this file as I don't understand why these changes have been made.